visualizer/parseTree.js

```diff
@@ -609,15 +609,15 @@
           visibleChoice: isVisibleChoice(node, parent)
         });
 
+        ohmEditor.parseTree.emit('create:traceElement', el, rootTrace, node);
```

Comment:
I moved this below the `isCollapsed` stuff because it seems to me that we haven't really finished creating the trace element here. I think this event is more generally useful if the element is really "done" when we call it.

Is there a reason you need it be here instead?